2345/3/models/article_creater.rb

```diff
@@ -0,0 +1,28 @@
+class ArticleCreater
+  attr_reader :article
```

Comment:
You break Command Query separation principle:
when you say ArticleCreater#create - it is a command method - it COMMANDs to do smth
when you need article - it is a QUERY for a state of the object - so you should have 2 separate method for it 
(I use attr_reader to create this method see example below)
so you should split the #create method into 2 methods

```
class ArticleCreator
   attr_reader :options   #input
   attr_reader :article     #output

  # helper method not to break command/query separation principle 
  # you should replace all places where you use ArticleCreater.new(options).create to ArtcleCreator.create(options)
  def self.create(options) 
    article_creator = new(options)
    article_creator.create(options)
    article_creator.article
  end

  def initialize(options)
  end
 
  def create
    create_article
    create_comments
  end

  private

  def create_article
    @article = Article.create(link: @article_link, title: title, rating: article_rating)
  end

  def create_comments
    CommentsCreater.new(comments_with_rating, article).create
  end
end